### DISCUSSION TITLE
Extending mantine-sizing with additional sizes

### DISCUSSION THREAD

**Discussion by kylegrover** [mantinedev/mantine#5394] [Q&A]
Loving Mantine and finding that I rarely have to reach for magic numbers or custom css, but one of the most frequent exceptions to that rule is when I need a bit more spacing. Is there any way to add for instance custom '2xl' and '3xl' values to be usable in the `p=` and `m=` props of all Mantine components?

**Comment by rtivital** [comment]
Add values to theme.spacing, they must not start with numbers, use `xl2`, `xl3`. Then use them in your components.
